nixos/modules/programs/sway.nix

```diff
@@ -87,20 +89,35 @@ in {
       type = with types; listOf package;
       default = with pkgs; [
         swaylock swayidle alacritty dmenu
-        rxvt-unicode # For backward compatibility (old default terminal)
```

Comment:
Not sure if that requires a release notes entry. It's likely annoying if starting a terminal doesn't work anymore but Sway users will hopefully be skilled enough by figuring this out quickly (e.g. resorting to `dmenu`).

Reply:
Maybe we could add a wrapper that launchers alacritty but displays a warning?

Reply:
I thought about that but not sure if it's a good idea / if a nice implementation is possible (would have to be a graphical warning and then it get's hacky quickly). Do you have something concrete in mind?

Reply:
You could display a warning in the shell itself: `alacritty -e sh -c 'echo WARN; exec $SHELL'`

I don't know exactly how you'd make sure the real `rxvt-unicode` overrides this fake rxvt-unicode.

Reply:
> You could display a warning in the shell itself: `alacritty -e sh -c 'echo WARN; exec $SHELL'`

That looks and sound super hacky to me (haven't tried if it even works or if `exec $SHELL` resets the output).
Users could be very confused by this.
And, while I haven't thought about that yet, launching Alacritty for users that expect rxvt-unicode also sounds wrong.
Anyway, if someone ACKs that I'd be ok with that but to me it doesn't really sound like a good idea (the intentions are good but I feel like I'd be super confused).

> I don't know exactly how you'd make sure the real rxvt-unicode overrides this fake rxvt-unicode.

There's the `lowPrio` function ( `meta.priority`) that should be able to handle that.

Reply:
Users can easily switch back to a tty to install a terminal if they forgot to do so in the first place. Shipping an Xorg terminal wrapper by default on sway seems backwards.

Also, wouldn't it be sensible to go with wofi until we have wmenu? Scaling of Xorg apps can be pretty bad.

Reply:
Please no, `wofi` on nixpkgs master segfaults when I run it.

Reply:
Great, can you get a backtrace and open an issue?

Reply:
> Either way we should add a release notes entry, so people will not be too surprised about this.

How does #127927 look?

> Also, wouldn't it be sensible to go with wofi until we have wmenu?

Our `programs.sway.extraPackages` is "unopinionated", we're simply sticking to the upstream defaults (i.e. Drew's opinion / default configuration). dmenu certainly isn't ideal but is seems like there's still no good enough replacement for it: https://github.com/swaywm/sway/issues/5112

Reply:
> Great, can you get a backtrace and open an issue?

The opposite of great. I was under the impression that we don't push broken packages to solicit bug reports.

Reply:
Great(, that this problem is so reproducible for you), it would probably help you and others if we  could report this issue upstream, hopefully with a backtrace (`coredumpctl debug`).